apps/src/lib/node/ledger/shell/vote_extensions.rs

```diff
@@ -26,7 +26,6 @@ const VALIDATOR_EXPECT_MSG: &str = "Only validators receive this method call.";
 pub enum VoteExtensionError {
     #[error("The vote extension was issued at block height 0.")]
     IssuedAtGenesis,
-    #[cfg(feature = "abcipp")]
     #[error(
         "The vote extension has an unexpected sequence number (e.g. block \
          height)."
```

Comment:
I think I chose this name because I was trying to be generic over invalid epochs or block numbers. might just rename it to `UnexpectedBlockNumber`, since we abandoned epochs as nonces in validator set updates